VideoWeb/VideoWeb/ClientApp/src/assets/i18n/cy.json

```diff
@@ -1148,7 +1148,7 @@
   "quick-participant-errors": {
     "invalid-page": {
       "heading": "Does dim modd cydnabod y ddolen rydych chi wedi'i defnyddio",
-      "body": "Gwiriwch y ddolen a anfonwyd atoch. Os nad yw'n gweithio o hyd, ffoniwch 0300 303 0655 i siarad â swyddog gwrandawiadau fideo."
+      "body": "Gwiriwch y ddolen a anfonwyd atoch. Os nad yw'n gweithio o hyd, ffoniwch 0300 790 6234 i siarad â swyddog gwrandawiadau fideo."
```

Comment:
@krishnapottekkat-hmcts Sure, it would be appropriate to do so if the file was a component's view template but it isn't.  This file is a json type and is used for scottish translation.